source/_patterns/basics/dropdowns/00-dropdown.hbs

```diff
@@ -1,7 +1,11 @@
-<div class="dropdown {{ dropdownState }} {{#if pullright}}pull-right{{/if}}" aria-labelledby="{{ dropdownAriaLabelledby }}">
-  <button type="button" class="btn {{ btnClass }} dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-    {{ dropdownText }}
-  </button>
+<div class="dropdown {{ dropdownState }} {{#if isKebab}}pf-dropdown__kebab{{/if}} {{#if pullright}}pull-right{{/if}}" aria-labelledby="{{ dropdownAriaLabelledby }}">
+  {{#if isKebab}}
+    {{> basics-icon-button btnText="" btnClass="btn-link" iconFamily="fa" iconName="fa-ellipsis-v" iconModifier="" ariaLabeledBy="Kebab"}}
+  {{else}}
```

Comment:
I think this is a good solution. Let's see how the kebab evolves, it's an element that is reused in many places and we might have to come back and accommodate this as we find new challenges with it.
I'd like to know what @srambach thinks